src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -166,6 +167,7 @@ const DiscoverChartList = ({ isSuperAdmin }: { isSuperAdmin: boolean }) => {
             chartRepos.sort((a, b) => a['name'].localeCompare(b['name']))
             setChartLists(chartRepos)
             setFilteredChartList(chartRepos)
+            setChartActiveMap(chartRepos.reduce((acc, curr) => ({ ...acc, [curr.name]: curr.active }), {}))
```

Comment:
Suggestion: Imo, this would be way more optimized:
```suggestion
            setChartActiveMap(chartRepos.reduce((acc, curr) => {
                acc[curr.name] = curr.active
                return acc
            }, {}))
```